### DISCUSSION TITLE
How to refer a file under patches which is exists in another path or folder

### DISCUSSION THREAD

**Discussion by LAVA-KUMAR** [kubernetes-sigs/kustomize#5417] [Q&A] [answered]
Hi team , can anyone help with this following scenario :
I want to refer a file(here the file  exists inside another folder) under patches  in the kustomisation.yaml . how to achieve this ?

Let me explain the flow :
assume I have a file called demo.yaml under k8s/base/deployment ( this is base )
assume the kustomisation.yaml I am talking about is present inside the k8s/overlay/deployment
Now in the above overlay kustomisation.yaml I want to refer demo.yaml under patches something like this :

apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
patches:
 -  ../../base/deployment/demo.yaml
  
if I do kubectl kustomize on the overlay I am getting error . can anyone explain how can I refer demo.yaml file under patches here . Thanks in advance .

[a comment by an automated account is not shown]

**Comment by seh** [comment, marked as answer]
Try setting the `--load-restrictor` flag to "'LoadRestrictionsNone" in order to relax the default restriction that all referenced files must be dominated by the _kustomization_ root directory.

Also pertinent is the recent #5216.

**Comment by annasong20** [comment]
@LAVA-KUMAR, does @seh's response answer your question?

As side notes, we usually recommend support questions be asked in the Kustomize Slack channel
https://kubernetes.slack.com/archives/C9A5ALABG. @stormqueen1990 also suggested we can additionally open up GitHub discussions on our repo. Converting this issue into a discussion.

**Comment by annasong20** [comment]
@sarab97 What do you think of opening GitHub discussions on the Kustomize repo? It'd be great if we could formalize this new best practice, either via flows or documentation.
